examples/example_B1855.py

```diff
@@ -15,10 +13,10 @@
 timfile = os.path.join(datadir, 'B1855+09_NANOGrav_dfg+12.tim')
 
 # libstempo calculation
-print "tempo2 calculation"
+print("tempo2 calculation")
```

Comment:
How does this work without an import of `print_function` in Python 2?

Reply:
In Python 2, what happens is that the `()` are ignored as a redundant set of parentheses:
```
$ python2
>>> print('spam')
spam
>>> print('spam', 'ham')
('spam', 'ham')
>>> print('spam', )
('spam',)
>>> ('spam')
'spam'
```
I can add the `from __future__ import print_function` line at the top if you want.